k2/csrc/ragged.h

```diff
@@ -98,6 +98,13 @@ class RaggedShape {
     return axes_[axis - 1].row_splits;
   }
 
+  const Array1<int32_t> &RowSplits(int32_t axis) const {
```

Comment:
no need for a separate one for const and non-const: just mark the other one const.

Reply:
> k2/csrc/ragged.h(98): error: qualifiers dropped in binding reference of type "k2::Array1<int32_t> &" to initializer
of type "const k2::Array1<int32_t>"

The function returns a reference and the compiler will complain.